src/res/translation/wininstaller/de.nsi

```diff
@@ -22,10 +22,10 @@ LangString ASIO_DRIVER_HEADER ${LANG_GERMAN} \
      "ASIO Treiber"
 
 LangString ASIO_DRIVER_SUB ${LANG_GERMAN} \
-     "Jamulus benötigt einen ASIO Treiber"
+     "${APP_NAME} benötigt einen ASIO Treiber"
```

Comment:
Should have been a separate PR ;-)

Reply:
Do you want me to undo and create a seperate PR for that? DE is not the only language in which I have updated that.

Reply:
No. It’s fine like that. Just for future notice it would be better to have one PR per change.